File: Baekjoon_Online_Judge/NO_16968/16968.cpp
#include <iostream>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 27                   | 
 | 16968 차량 번호판 1        | 
 ---------------------------- 
*/

int main(){
    string form;

    cin >> form;

    int len = form.length();
    int save[len], res = 1;

    for(int i = 0; i < len; i ++){
        if(form[i] == 'd') save[i] = 10;
        else save[i] = 26;
    }

    for(int i = 0; i < len - 1; i ++){
        if(form[i] == form[i + 1]) save[i] -= 1;
    }

    for(int i = 0; i < len; i ++) res *= save[i];

    cout << res;
    return 0;
}

File: Baekjoon_Online_Judge/NO_16968/16968.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 27                   | #
# | 16968 차량 번호판 1        | #
# ---------------------------- #


if __name__ == "__main__":
    form = r_input().rstrip()

    length = len(form)
    save = [0] * length

    for ind, val in enumerate(form):
        if val == 'd':
            save[ind] = 10
        
        else:
            save[ind] = 26
        
    for ind in range(length - 1):
        if save[ind] == save[ind + 1]:
            save[ind] -= 1
    
    res = 1

    for val in save:
        res *= val
    
    print(res)
